openedx/core/djangoapps/discussions/docs/decisions/0008-reported-content-email-notifications.rst

```diff
@@ -0,0 +1,77 @@
+Reported content email notifications for moderators
+===================================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+We intend to provider an option for quick visibility of reported content for
+moderators. We intend to implement this by building an option to send out an
+email to discussion moderators when a content (post/response/comment) is
+reported. This email will be sent out when a content is reported, and stays
+reported for the next 2 minutes.
+
+We intend to send the email to all roles that have discussion
+moderation privileges, namely Discussion Admin, Discussion Moderator,
+Community TA and Group Community TA.
+
+Group community TAs will receive emails for reported content in their own
+cohort.
+
+A toggle for this setting will be shown on the edX discussions configuration
+page. This toggle can be turned on or off at any point in time during the
+course run.
+
+
+Requirements
+------------
+
+We need to have the ability to enable/disable email notifications for reported
+content for a course.
+
+When course content is reported, we have to send out an email to moderators 2
```

Comment:
When a course content is reported the email needs to be sent only if the content remains reported for the next 2 minutes.